cmd/infra/powervs/create.go

```diff
@@ -69,6 +70,11 @@ const (
 	powerVsService  = "powervs"
 	vpcService      = "vpc"
 	platformService = "platform"
+
+	// Secret suffix
+	kubeCloudControllerManagerCredSuffix = "-cloud-ctrl-creds"
```

Comment:
where are these prefix coming from? can we be consistent with https://github.com/openshift/hypershift/blob/ded35395d9b2416c1d0a1c47596e7a50903a66ab/hypershift-operator/controllers/hostedcluster/internal/platform/aws/aws.go#L343-L366?

Reply:
These var are used to create serviceID secrets in IBM Cloud, renamed them similar to the creds mentioned, ptal.